### ISSUE TITLE
"Make available offline" option on FileListing

### ISSUE THREAD

**Issue by TristanWright** [Kitware/HPCCloud#378]
select multiple files/folder and present an action which will copy them off the cluster and into Girder.

**Comment by TristanWright** [comment]
For this action, it copies the item from the cluster to a new item on the girder server? so the steps: 

With the external item id being `f2`
- `POST /item`: create new item `f1` in girder
- `POST /item/f2/copy`: copy contents of external item `f2` to item `f1`

Or along those lines? Also is there a progress event we can listen to for this?

**Comment by cjh1** [comment]
@TristanWright So the idea as you say is to copy the data from the cluster into the girder servers assetstore. I think the endpoint we want is:

```
PUT /file/{id}/move
```

This atomically move the file from the current assetstore ( on the cluster ) into another one ( for example on the local filesystem ).

Not sure if the request returns something that can be used to track progress. We would have to experiment to see.

**Comment by TristanWright** [comment]
Any word on how to get the destination assetstoreId param in

```
PUT /file/{id}/move?assetstoreId=[....]
```

**Comment by cjh1** [comment]
I will send something to the Girder list ....

**Comment by TristanWright** [comment]
closed with #561
